.github/workflows/build-solution.yml

```diff
@@ -67,16 +68,8 @@ jobs:
           sudo apt-get update
           sudo apt-get install -y libpcap-dev amazon-ecr-credential-helper
 
-      - name: Install docker-credential-ecr-login
-        run: |
-          # Download the latest release of docker-credential-ecr-login
-          aws s3 cp s3://fortanix-internal-artifact-repository/other-packages/docker-credential-ecr-login .
-
-          # Make it executable
-          chmod +x docker-credential-ecr-login
-
-          # Move it to a directory in the PATH
-          sudo mv docker-credential-ecr-login /usr/local/bin/
+          # Setup docker-credential-ecr-login
```

Comment:
Can we add a more relevant comment here? Why are we moving the binary to /usr/local/bin from /usr/bin? `/usr/bin` is already part of the $PATH.

Reply:
It's because of this rule https://github.com/fortanix/salmiac/blob/7f320a37b9f84e5a288c236cff707dd8ffdc43c3/make/salmiac-tests-container.make#L130.